### PR TITLE
Prevent advanced gas input arrows from setting value to < 0

### PR DESCRIPTION
**Pull request by danjm**
fixes https://sentry.io/organizations/metamask/issues/919797069/?project=273505&query=is%3Aunresolved+release%3A6.2.0&statsPeriod=14d&utc=false

### CODE REVIEW

File: ui/app/components/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js [hunk 1]
```diff
@@ -80,8 +80,23 @@ export default class AdvancedTabContent extends Component {
...
+          >
+            <i className="fa fa-sm fa-angle-up" />
+          </div>
+          <div
+            className="advanced-gas-inputs__gas-edit-row__input-arrows__i-wrap"
+            onClick={() => {
+              const newValue = value >= 1
+                ? value - 1
+                : 0
+              onChange(newValue)
```
Comment (whymarrh) [inline ui/app/components/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js]: Could we instead write `onChange(Math.max(value - 1, 0))`?

File: ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js [hunk 2]
```diff
@@ -88,8 +88,23 @@ export default class AdvancedTabContent extends Component {
...
+          >
+            <i className="fa fa-sm fa-angle-up" />
+          </div>
+          <div
+            className="advanced-tab__gas-edit-row__input-arrows__i-wrap"
+            onClick={() => {
+              const newValue = value >= 1
+                ? value - 1
+                : 0
+              onChange(newValue)
```
Comment (whymarrh) [inline ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js]: Could we instead write `onChange(Math.max(value - 1, 0))`?

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: ui/app/components/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js (hunk 1 above)
Comment (danjm) [reply ui/app/components/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js]: Nice. Done in 1296bcc00

File: ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js (hunk 2 above)
Comment (danjm) [reply ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js]: 1296bcc00

**Review by whymarrh — APPROVED** [review]

**Pull request merged**